test/parallel/test-http-remove-header-stays-removed.js

```diff
@@ -54,13 +54,14 @@ server.listen(0, function() {
     res.setEncoding('ascii');
     res.on('data', function(chunk) {
       response += chunk;
+      if (response?.toString() === 'beep boop\n') {
```

Comment:
```suggestion
      if (response === 'beep boop\n') {
```

Response is always a string.